planetiler-basemap/src/main/java/com/onthegomap/planetiler/basemap/Generate.java

```diff
@@ -128,8 +128,10 @@ static JsonNode parseYaml(String string) {
   public static void main(String[] args) throws IOException {
     Arguments arguments = Arguments.fromArgsOrConfigFile(args);
     PlanetilerConfig planetilerConfig = PlanetilerConfig.from(arguments);
-    String tag = arguments.getString("tag", "openmaptiles tag to use", "v3.12.2");
-    String base = "https://raw.githubusercontent.com/openmaptiles/openmaptiles/" + tag + "/";
+    String tag = arguments.getString("tag", "openmaptiles tag to use", "v3.13");
+    String baseUrl = arguments.getString("base-url", "the url used to download the openmaptiles.yml",
+      "https://raw.githubusercontent.com/openmaptiles/openmaptiles/");
+    String base = baseUrl + tag + "/";
```

Comment:
what do you think about providing a base URL and if it's just a tag then assume it's based off of the openmaptiles repo?

Reply:
Ok sure, that's a good idea. So we replace tag and base-url with something like `source` or `base`?

Reply:
Actually nevermind I think it's probably better as-is. I always get mixed up on what the URL pattern for raw.githubusercontent is, so seeing that example string there that you can override would be easier to use.